engine/runtime/src/main/java/org/enso/interpreter/runtime/data/atom/AtomConstructor.java

```diff
@@ -278,8 +384,9 @@ public String toString() {
    *
    * @return the constructor function of this constructor.
    */
+  @TruffleBoundary
```

Comment:
This is slowing execution down. 

![obrazek](https://github.com/user-attachments/assets/ecb32087-b2b8-48b8-9225-ebc0d295b589)

This method needs to be inlined, it cannot be `@TruffleBoundary`.